packages/create-config/index.js

```diff
@@ -19,16 +19,29 @@ const getPackageManager = () => {
   return 'npm';
 };
 
+const getWorkspaceFlag = (pm) => {
+  if(pm === 'pnpm') {
+    return fileExists('pnpm-workspace.yaml') ? '-w' : undefined;
+  } else if(pm === 'yarn') {
+    const packageJsonPath = path.join(process.cwd(), 'package.json');
+    if (fileExists(packageJsonPath)) {
```

Comment:
This check is redundant.

Reply:
You are right. The readFileSync call will throw an error anyway.